lib/signaling/v2/peerconnectionmanager.js

```diff
@@ -25,12 +26,31 @@ function PeerConnectionManager(iceServerSource, options) {
   QueueingEventEmitter.call(this);
 
   options = Object.assign({
-    audioContext: audioContext,
+    AudioContextFactory: require('../../webaudio/audiocontext'),
```

Comment:
@markandrus Nitpick: Since this is a singleton instance, can we camel-case the name to `audioContextFactory`.